ejb3/src/main/java/org/jboss/as/ejb3/remote/AssociationImpl.java

```diff
@@ -393,7 +434,7 @@ public void removedEntries(final Map<String, List<ClientMapping>> removed) {
         }
     }
 
-    static Object invokeMethod(final ComponentView componentView, final Method method, final InvocationRequest incomingInvocation, final InvocationRequest.Resolved content, final CancellationFlag cancellationFlag) throws Exception {
+    static ResultAndContextData invokeMethod(final ComponentView componentView, final Method method, final InvocationRequest incomingInvocation, final InvocationRequest.Resolved content, final CancellationFlag cancellationFlag) throws Exception {
```

Comment:
@dmlloyd yeah, passing it as a parameter seems more common in the code. No complains from me, I will rework that.